pom.xml

```diff
@@ -1174,6 +1184,13 @@
                             <goals>
                                 <goal>compile</goal>
                             </goals>
+                            <configuration>
+                                <sourceDirs>
+                                    <source>src/main/java</source>
+                                    <source>src/main/kotlin</source>
+                                    <source>src/main/resources</source>
```

Comment:
It looks weird that the "resources" directory should be considered a source directory. Kotlin should not compile files from there. Does it still work if you remove this?